tools/utils.py

```diff
@@ -21,13 +21,23 @@ def wait_for_droplet_creation(droplet):
         print(f'   Exception: {err}')
         raise
 
+def wait_for_droplet_ip(droplet, timeout_seconds=None):
```

Comment:
timeout should have a default value to avoid an infinite while if none has been provided.
for example: `1000`

Reply:
Yes I forgot it